src/com/carolinarollergirls/scoreboard/rules/Rule.java

```diff
@@ -1,35 +1,48 @@
+    STOP_PC_ON_OTO(new BooleanRule("Timeout.StopPeriodClockOnOTO", "Stop the period clock on official timeouts?", false, "True", "False")),
+    STOP_PC_ON_TTO(new BooleanRule("Timeout.StopPeriodClockOnTTO", "Stop the period clock on team timeouts?", false, "True", "False")),
+    STOP_PC_ON_OR(new BooleanRule("Timeout.StopPeriodClockOnOR", "Stop the period clock on official reviews?", false, "True", "False")),
+    STOP_PC_AFTER_TO_DURATION(new TimeRule("Timeout.StopPeriodClockAfterTODuration", "Stop the period clock, if a timeout lasts longer than this time. Set to a high value to disable.", "60:00")),
+
+    INTERMISSION_DURATIONS(new StringRule("Intermission.Durations", "List of the duration of intermissions as they appear in the game, separated by commas.", "15:00,60:00")),
+    INTERMISSION_DIRECTION(new BooleanRule("Intermission.ClockDirection", "Which way should the intermission clock count?", true, "Count Down", "Count Up")),
+
+    AUTO_START(new BooleanRule("Automate.AutoStart", "When the Linup time plus AutoStartBuffer has elapsed start a Jam or Timeout as defined below. Jam/Timeout/Period Clocks will be adjusted by the buffer time. This only works if the lineup clock is counting up.", false, "Enabled", "Disabled")),
+    AUTO_START_BUFFER(new TimeRule("Automate.AutoStartType", "How long to wait after end of lineup before auto start is triggered.", "0:02")),
+    AUTO_START_JAM(new BooleanRule("Automate.AutoStartBuffer", "What to start after lineup is up", false, "Jam", "Timeout")),
+    AUTO_END_JAM(new BooleanRule("Automate.AutoEndJam", "End a jam, when the jam clock has run down", true, "Enabled", "Disabled")),
+    AUTO_END_TTO(new BooleanRule("Automate.AutoEndTTO", "End a team timeout, after it's defined duration has elapsed", false, "Enabled", "Disabled")),
+
+    NUMBER_TIMEOUTS(new IntegerRule("Team.Timeouts", "How many timeouts each team is granted per game or period", 3)),
+    TIMEOUTS_PER_PERIOD(new BooleanRule("Team.TimeoutsPer", "Are timeouts granted per period or per game?", false, "Period", "Game")),
+    NUMBER_REVIEWS(new IntegerRule("Team.OfficialReviews", "How many official reviews each team is granted per game or period", 1)),
+    REVIEWS_PER_PERIOD(new BooleanRule("Team.OfficialReviewsPer", "Are official reviews granted per period or per game?", true, "Period", "Game")),
+
+    PENALTIES_FILE(new StringRule("Penalties.DefinitionFile", "File that contains the penalty code definitions to be used", "/config/penalties/wftda2018.json")),
+    FO_LIMIT(new IntegerRule("Penalties.NumberToFoulout", "After how many penalties a skater has fouled out of the game. Note that the software currently does not support more than 9 penalties per skater.", 7));
+
+    private Rule(AbstractRule r) { rule = r; }
+    
+    public AbstractRule getRule() { return rule; }
+    public String toString() {return rule.getFullName(); }
```

Comment:
I'd expect toString to return all the information about a rule, not just one field. I think getFullName should be used directly where this is needed.

We could also make it getName now.